cluster-http/src/main/resources/reference.conf

```diff
@@ -1,106 +1,18 @@
-    # If undefined, the name will be extracted from the ActorSystem name
-    service-name = null
-
-    # Added as suffix to the service-name to build the effective-service name used in the contact-point service lookups
-    # If undefined, nothing will be appended to the service-name.
-    #
-    # Examples, set this to:
-    # "default.svc.cluster.local" or "my-namespace.svc.cluster.local" for kubernetes clusters.
-    service-namespace = null
-
-    # The effective service name is the exact string that will be used to perform service discovery
-    # usually, this means the service-name being suffixed with an additional namespace (e.g. "name.default"
-    effective-name = null
-
-    # Config path of discovery method to be used to locate the initial contact points.
-    # It must be a fully qualified config path to the discovery's config section.
-    discovery-method = akka.discovery.akka-dns
-
-    # Amount of time for which a discovery observation must remain "stable"
-    # (i.e. not change list of discovered contact-points) before a join decision can be made.
-    # This is done to decrease the likelyhood of performing decisions on fluctuating observations.
-    stable-margin = 3 seconds
-
-    # Interval at which service discovery will be polled in search for new contact-points
-    interval = 1 second
-
-    # The smallest number of contact points that need to be discovered before the bootstrap process can start.
-    # For optimal safety during cluster formation, you may want to set these value to the number of initial
-    # nodes that you know will participate in the cluster (e.g. the value of `spec.replicas` as set in your kubernetes config.
-    required-contact-point-nr = 2
-
-    # Timeout for getting a reply from the service-discovery subsystem
-    resolve-timeout = 3 seconds
-
-  }
-
-  # Configured how we communicate with the contact point once it is discovered
-  contact-point {
-
-    # If no port is discovered along with the host/ip of a contact point this port will be used as fallback
-    fallback-port = 8558 # port pun, it "complements" 2552 which is often used for Akka remoting
-
-    # If no seed nodes are discovered from a given contact-point by this time
-    # it will assume that the the observation is "stable" (i.e. will not change),
-    # and a new cluster may need to be formed. In response to this the main bootstrap
-    # coordinating process may decide to join itself, or keep waiting for discovering of a seed node.
-    no-seeds-stable-margin = 5 seconds
-
-    # Interval at which contact points should be polled
-    # the effective interval used is this value plus the same value multiplied by the jitter value
-    probe-interval = 1 second
-
-    # Max amount of jitter to be added on retries
-    probe-interval-jitter = 0.2
-
-    # Probe will be failed if it doesn't return in this amount of time
-    probe-timeout = 1 second
+  cluster.http {
```

Comment:
Any point in keeping this block when it's anyways empty?

Reply:
Yeah I wanted to keep it since this is the place to configure this module. In case someone contributes etc